styles.css

```diff
@@ -786,3 +786,70 @@ button {
     width: 100%;
   }
 }
+
+@media only screen and (min-width: 768px) {
+  html {
+    overflow-x: hidden;
+    width: 100%;
+  }
+
+  .works {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .headline {
+    background-image: url(icons/Header-llustration-desktop.svg);
+    background-position: initial;
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .nav {
+    visibility: hidden;
+  }
+
+  .container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: absolute;
+    margin: 15%;
+  }
+
+  .headline p {
+    align-self: center;
+    position: relative;
+    margin: 25%;
+  }
+
+  .contact1 {
+    position: absolute;
+    margin-top: 40%;
+  }
+
+  .contact1 ul li {
+    width: 30%;
+    height: 10%;
+    border-radius: 0%;
+  }
+
+ .contact1 ul li img {
+    width: 12px;
+    margin: 5px;
+  }
```

Comment:
to make the website more responsive it is advise to use % for margins and paddings, and for height instead of px vh, for width instead of px to use vw :)

Reply:
Thanks, Noted!